README.md

```diff
@@ -102,21 +98,21 @@ Let's posit that code complexity is:
 
 Where "things" are:
 
-1. **data** (ex. a user address; an email's contents; an object's state)
+1. **data**
 2. **the outside world** (ex. availability of a network; the current time)
 3. **logical decisions** (ex. email user?; print message?; render page?)
 
 Naturally, these things are sources of complexity:
 
-1. more data (especially data which can change or may not exist)
-2. more interaction with the outside world (especially parts we do not control)
-3. more decisions (especially ones which do not handle every case)
+1. changing data
+2. more of the outside world (especially parts we do not control)
+3. more branch logic (especially when non-exhaustive)
+4. optional data (nil), which necessitates more branch logic
 
 (\#1 and \#2 are collectively referred to as **side effects**.)
 
-Simple software minimizes its exposure to complexity. This is why the latest
-iteration of our code is objectively simpler. Again, with the above lists in
-consideration:
+Simple software minimizes its exposure to complexity. Again, with the above
```

Comment:
Based on your definition of complexity [here](https://github.com/dailykos/straitjacket/pull/2/files#diff-04c6e90faac2675aa89e2176d2eec7d8R97) I posit that the root of the problem is not complexity.

> Code we must write to account for all possible states of things.

This suggests that building software that has more functionality makes the code inherently more complex. I have seen many counter examples to this in the real world on both sides.

I suggest considering the root of the problem here with a precise definition. :)